resources/js/Pages/Team/Partials/Members.vue

```diff
@@ -151,7 +152,8 @@ export default {
 
     // when a team lead is set, we must react to the event emitted by the
     // TeamLead.vue component and add the team lead to the list of employees
-    this.$root.$on('lead-set', employee => {
+    const emitter = new Emitter();
+    emitter.on('lead-set', employee => {
```

Comment:
This doesn't work anymore. The team lead is not added to the list of employees when he's not part of the team.

Reply:
Fixed now